### DISCUSSION TITLE
Lets create a js library for lively wallpapers

### DISCUSSION THREAD

**Discussion by 1j4d5** [rocksdanister/lively#2045] [General]
Give me some ideas on functions to add

**Comment by alany08** [comment]
Dynamic realistic sky rendering based on time of day, weather (rainy, cloudy, windy, etc.)

**Comment by EthanHuangSSZ136** [comment]
I also need some suggestions, I used SlidingClock as the wallpaper, but the content on the right side was a little empty, so I made a right-hand column, now I have added the weather display, I want to add some more content, but I don't know what to add
![20240206143525](https://github.com/rocksdanister/lively/assets/155513655/4ff6b925-6a49-4f36-9181-5e377189a045)
